judge/views/ticket.py

```diff
@@ -77,6 +81,11 @@ def form_valid(self, form):
                 'message': message.id, 'user': ticket.user_id,
                 'assignees': list(ticket.assignees.values_list('id', flat=True)),
             })
+        if isinstance(ticket.linked_item, Problem):
+            logger.info("New ticket for {problem}: {url}".format({
```

Comment:
Single quotes. Also, use `logging`'s standard formatting. `logger.info('New ticket for %s: %s', problem_code, url)`